ggml/src/ggml.c

```diff
@@ -18047,6 +18388,8 @@ static void ggml_compute_backward(struct ggml_context * ctx, struct ggml_tensor
             } break;
```

Comment:
`GGML_UNARY_OP_EXP` should probably be handled explicitly here. The backward computation is the same as the forward.

Reply:
```
case GGML_UNARY_OP_EXP:
    {
        if (src0->grad) {
            src0->grad = ggml_add_or_set(ctx,
                src0->grad,
                ggml_mul(ctx, tensor, tensor->grad),
                zero_table);
             }
    } break;
```
Like this?

Reply:
Looks correct to me 👍